cli/verbs.go

```diff
@@ -35,6 +36,32 @@ func run(requestor string, transport transport.Transport, stdout io.Writer) erro
 	var err error
 	switch verb {
 	case "start":
+
+		//get the environment variables
+		binaryURL := os.Getenv("FIRMWARECI_BINURL")
+
+		//if the variable is set, download the binary
+		//the url is presumed to be a url and from a
+		//trusted domain
+		if binaryURL != "" {
+			//the file name is hardcoded atm
+			file, err := os.Create("/root/assets/TEST_BIN")
+			if err != nil {
+				return err
+			}
+			defer file.Close()
+
+			resp, err := http.Get(binaryURL)
```

Comment:
This assumes it's always http - correct? IIRC we stated multiple download methods?